build.gradle.kts

```diff
@@ -114,29 +160,26 @@ val downgradeJar17 = tasks.register<DowngradeJar>("downgradeJar17") {
 }
 
 tasks.jar {
-    isPreserveFileTimestamps = false
-    isReproducibleFileOrder = true
```

Comment:
Why remove these settings?

Reply:
it was moved down to a block later `tasks.withType<AbstractArchiveTask>`
not really necessary but good grade practice